src/pages/CommunityResourcesPage/ui/CommunityResourcesPage.tsx

```diff
@@ -2,7 +2,7 @@ import React, { useState } from "react";
 import ResourceCard from "../../../shared/ui/ResourceCard/ResourceCard";
 import { resources } from "./constants";
 import styles from "./CommunityResourcesPage.module.css";
-import Button from "../../Pomodoro/components/Button/Button";
+import Button from "../../../shared/ui/Button/Button.tsx";
```

Comment:
Here Button properties need also to be updated, I will do this in my PR though, since I can't deploy the app without fixing it :)